src/util.h

```diff
@@ -768,16 +768,19 @@ class CHostAddress
 
     CHostAddress() :
         InetAddr ( static_cast<quint32> ( 0 ) ),
-        iPort ( 0 ) {}
+        iPort ( 0 ),
+        LocalAddr ( static_cast<quint32> ( 0 ) ) {}
 
     CHostAddress ( const QHostAddress NInetAddr,
                    const quint16      iNPort ) :
         InetAddr ( NInetAddr ),
-        iPort    ( iNPort ) {}
+        iPort    ( iNPort ),
+        LocalAddr ( static_cast<quint32> ( 0 )) {}
 
     CHostAddress ( const CHostAddress& NHAddr ) :
         InetAddr ( NHAddr.InetAddr ),
-        iPort    ( NHAddr.iPort ) {}
+        iPort    ( NHAddr.iPort ),
+        LocalAddr ( static_cast<quint32> ( 0 )) {}
```

Comment:
Please follow standard alignment of code - even where that means adjusting existing code.